src/aid.rs

```diff
@@ -54,20 +54,16 @@ pub enum Category {
 
 impl core::fmt::Debug for Aid {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
-        if self.len <= self.truncated_len {
-            f.write_fmt(format_args!(
-                "'{} {}'",
-                hexstr!(&self.bytes[..5]),
-                hexstr!(&self.bytes[5..self.len as _])
-            ))
-        } else {
-            f.write_fmt(format_args!(
-                "'{} {} {}'",
-                hexstr!(&self.bytes[..5]),
-                hexstr!(&self.bytes[5..self.truncated_len as _]),
-                hexstr!(&self.bytes[self.truncated_len as _..self.len as _])
-            ))
+        f.write_str("'")?;
+        for b in &self.bytes[..5] {
+            f.write_fmt(format_args!("{:02X}", b))?;
         }
+        f.write_str(" ")?;
+        for b in &self.bytes[5..self.len as _] {
+            f.write_fmt(format_args!("{:02X}", b))?;
+        }
+        f.write_str("'")?;
+        Ok(())
```

Comment:
Please keep the additional space in the case where `self.truncated_len < self.len`.